samples/kafka/akka/README.md

```diff
@@ -1,8 +1,8 @@
-# Using Akka Streams with Event Hubs for Kafka Ecosystem
+# Using Akka Streams with Event Hubs for Apache Kafka Ecosystem
 
-One of the key benefits of using Apache Kafka is the ecosystem of frameworks it can connect to. Kafka enabled Event Hubs allow users to combine the flexibility of the Kafka ecosystem with the scalability, consistency, and support of the Azure ecosystem without having to manage on prem clusters or resources - it's the best of both worlds!
+One of the key benefits of using Apache Kafka is the ecosystem of frameworks it can connect to. Kafka-enabled Event Hubs allow users to combine the flexibility of the Kafka ecosystem with the scalability, consistency, and support of the Azure ecosystem without having to manage on prem clusters or resources - it's the best of both worlds!
```

Comment:
> One of the key benefits of using Apache Kafka is the ecosystem of frameworks it can connect to. 

One key benefit of using Apache Kafka is the vast client ecosystem available for users.

Reply:
This line seems to appear in multiple places, so update it everywhere (if you agree that it should be changed)

Reply:
I don't want to mix singular and plural references to the ecosystem (Kafka's ecosystem and Kafka's ecosystems), so I'm going to stick with something similar to what I had

Reply:
k, the original has `ecosystem` though. how about:

One key benefit of using Apache Kafka is the many ecosystems it can connect to.